packages/delayedInbox-l2msg/README.md

```diff
@@ -1,41 +1,42 @@
-# delayedInbox-l2msg Tutorial
+# Tutorial: send a message from the parent chain
```

Comment:
We can use `send a signed transaction from the parent chain` for better because other ways like retryables can also send a message